src/enumerable.cr

```diff
@@ -1291,4 +1291,17 @@ module Enumerable(T)
       hash[item[0]] = item[1]
     end
   end
+
+  # Creates a `Hash` out of an returned from a *block* Enumerable, where each
+  # element is a 2 element structure (for instance a `Tuple` or an `Array`).
```

Comment:
The docs are now outdated. Also it reads a bit strange "out of an returned"

Reply:
> Creates a `Hash` out of `Tuple` pairs (key, value) returned from *block*.

sounds good?

Reply:
Yes! Maybe saying that the first element of the tuple will be used as a key, and the second as a value, though I think this will be clear from the example. Alternatively check the Ruby docs (but they will probably mention Array).

Reply:
Fixed.